osu.Game/Localisation/OnlinePlayStrings.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class OnlinePlayStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.OnlinePlay";
+
+        /// <summary>
+        /// "This duration is only available for osu!supporters."
+        /// </summary>
+        public static LocalisableString SupporterOnlyDurationNotice => new TranslatableString(getKey(@"supporter_only_duration_notice"), @"This duration is only available for osu!supporters.");
```

Comment:
Might want to phrase this similar to the other case we already have:

https://github.com/ppy/osu/blob/de7f2a0bba6953b486689427198b0b1fd2804a30/osu.Game/Localisation/LeaderboardStrings.cs#L42-L45

Reply:
Maybe similar to the one in beatmap listing?
![osu!_6opU0y8Ola](https://github.com/ppy/osu/assets/35318437/8fe7b45b-c0d0-4c11-9886-bb3f72c4cca8)

`This duration requires an active osu!supporter tag.`